parts/js/admin.js

```diff
@@ -432,10 +432,13 @@
         if (!wp.autosave)
         {
           var editor = tinyMCE.activeEditor;
+          if(editor != null){
```

Comment:
Is this simply a falsy check? If so, let's just do `if(!editor)`. Otherwise, if we know what it will be then use the type comparison operator.

Reply:
im not frontend developer actually so i dont even know which state returned by tinymce, 

i thought that tinymce can be return false that is why me check null.

Reply:
@jason has this been tested?

Reply:
@kevinpmiller Looks like @sbruner did. I've never used VC, myself.